frontend/src/components/sharing/GroupAndRoleTable.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useEffect, useState} from "react";
+import {RootState} from "../../types/data";
+import {fetchDatasetGroupsAndRoles, fetchDatasetUsersAndRoles} from "../../actions/dataset";
+import {useDispatch, useSelector} from "react-redux";
+import {useParams} from "react-router-dom";
+import TableContainer from "@mui/material/TableContainer";
+import Paper from "@mui/material/Paper";
+import Table from "@mui/material/Table";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableCell from "@mui/material/TableCell";
+import TableBody from "@mui/material/TableBody";
+
+
+export const GroupAndRoleTable = (): JSX.Element => {
+
+	const {datasetId} = useParams<{ datasetId?: string }>();
+
+	const dispatch = useDispatch();
+
+	const getGroupsAndRoles = (datasetId: string | undefined) => dispatch(fetchDatasetGroupsAndRoles(datasetId));
+	const datasetGroupsAndRolesList = useSelector((state: RootState) => state.dataset.groupsAndRoles);
+	const [sharePaneOpen, setSharePaneOpen] = useState(false);
+
+	const handleShareClose = () => {
+        setSharePaneOpen(false);
+    }
+
+	useEffect(() => {
+		getGroupsAndRoles(datasetId);
+		console.log('groups and roles', datasetGroupsAndRolesList);
+	}, []);
+
+
+	const clickButton = () => {
+		// reset error message and close the error window
+		console.log('change role now');
+	}
+
+	return (
+		<TableContainer>
```

Comment:
Can we add a card, or white background similar to what we have in other tabs to make those tables consistent?